backend/endpoints/rom.py

```diff
@@ -10,47 +13,115 @@
 router = APIRouter()
 
 
+class RomSchema(BaseModel):
+    id: int
+
+    r_igdb_id: str
+    p_igdb_id: str
+    r_sgdb_id: str
+    p_sgdb_id: str
+
+    p_slug: str
+    p_name: str
+
+    file_name: str
+    file_name_no_tags: str
+    file_extension: str
+    file_path: str
+    file_size: float
+    file_size_units: str
+
+    r_name: str
+    r_slug: str
+
+    summary: str
+
+    path_cover_s: str
+    path_cover_l: str
+    has_cover: bool
+    url_cover: str
+
+    region: str
+    revision: str
+    tags: list
+
+    multi: bool
+    files: list
+
+    url_screenshots: list
+    path_screenshots: list
+
+    class Config(BaseModel.Config):
+        orm_mode = True
```

Comment:
At some point I'd like to move these model definitions into their own files, once we have more of them

Reply:
I was asking the same, didn't saw this comment. I agree with that, I guess for now we can leave it here and move it when it grows a little bit. For now it is manageable